Source/WebKit/WebProcess/WebCoreSupport/WebColorChooser.cpp

```diff
@@ -42,7 +43,10 @@ WebColorChooser::WebColorChooser(WebPage* page, ColorChooserClient* client, cons
     , m_page(page)
 {
     m_page->setActiveColorChooser(this);
-    WebProcess::singleton().parentProcessConnection()->send(Messages::WebPageProxy::ShowColorPicker(initialColor, client->elementRectRelativeToRootView(), client->suggestedColors()), m_page->identifier());
+    if (m_colorChooserClient->supportsAlpha())
+        WebProcess::singleton().parentProcessConnection()->send(Messages::WebPageProxy::ShowColorPickerEnhanced(initialColor, m_colorChooserClient->elementRectRelativeToRootView(), true, m_colorChooserClient->suggestedColors()), m_page->identifier());
+    else
+        WebProcess::singleton().parentProcessConnection()->send(Messages::WebPageProxy::ShowColorPicker(initialColor, m_colorChooserClient->elementRectRelativeToRootView(), m_colorChooserClient->suggestedColors()), m_page->identifier());
```

Comment:
(I think we can add the new alpha support argument to the existing `ShowColorPicker` IPC)